pageserver/src/metrics.rs

```diff
@@ -1252,6 +1252,18 @@ pub(crate) static WAL_REDO_RECORD_COUNTER: Lazy<IntCounter> = Lazy::new(|| {
     .unwrap()
 });
 
+pub(crate) static WAL_REDO_PROCESS_LAUNCH_DURATION_HISTOGRAM: Lazy<Histogram> = Lazy::new(|| {
+    register_histogram!(
+        "pageserver_wal_redo_process_launch_duration",
+        "Histogram of the duration of successful WalRedoProcess::launch calls",
+        // We'd expect `::launch()` to be significantly slower than an individual redo.
+        // But, they contribute to the same latency metric (walredo latency), so, use
+        // the same histogram for now.
```

Comment:
Hmm I had understood that nothing covers one walredo, checking...

Reply:
Does this mean `WAL_REDO_TIME` as "same latency metric (walredo latency)" or something else?

Note that `WAL_REDO_TIME` is used for N+1 -modal histogram because of how it's used also for the applies done in rust (`apply_batch_neon`). N is for unknown number of batch sizes.

Reply:
It is actuallly N+M where N is amount of apply_batch_neon and M is the amount of apply_batch_postgres records, nothing covers a single walrecord, nor can we easily get that (at best the avg).

```suggestion
```

rationale for removing:
- launch is not covered in WAL_REDO_TIME

Reply:
I guess I'll just remove this comment, no interest in reasoning about the existing walredo metrics (I never used them, I don't think anyone is actively looking at them, do you?)

Reply:
we do have them on the pageserver metrics and logs but they are quite ... nice to have but unsure if anyone is making any informed deductions about them.